extensions/sql/syntaxes/sql.tmLanguage.json

```diff
@@ -1,10 +1,10 @@
 {
 	"information_for_contributors": [
-		"This file has been converted from https://github.com/microsoft/vscode-mssql/blob/main/syntaxes/SQL.plist",
+		"This file has been converted from https://github.com/microsoft/vscode-mssql/blob/master/syntaxes/SQL.plist",
```

Comment:
this redirects to the main branch for me fwiw

Reply:
Yeah, that's intended. IIRC it's a bug with the script they use to generate this - it just assumes master branch for everything (even if it gets it from another branch such as main). I told Alex about this a long time back but seems like it's still around.